backend/src/services/AllowlistEntry.ts

```diff
@@ -27,7 +28,7 @@ function AllowlistEntryService(
   async function createAllowlistEntry({
     phoneNumber,
     destinationCountry,
-  }: Partial<AllowlistEntryEntity>) {
+  }: Pick<AllowlistEntryEntity, 'phoneNumber' | 'destinationCountry'>) {
```

Comment:
Update to `Pick` as `Partial` makes all attributes optional (ie `string | undefined`)